conf/application.conf

```diff
@@ -93,12 +93,12 @@ play.modules.enabled += "config.DIModule"
 api {
   # The status of the version of the API for the API Platform.
   1.0 {
-    status = "ALPHA"
-    endpoints.enabled = false
+    status = "BETA"
+    endpoints.enabled = true
```

Comment:
From confluence "V1 is to remain as false in production for the duration of the deprecation. " do we need to switch this to false?

Reply:
No because these values don't affect ET or Prod